MapboxNavigation/SystemSpeechSynthesizer.swift

```diff
@@ -0,0 +1,170 @@
+
+import AVFoundation
+import MapboxDirections
+import MapboxCoreNavigation
+import MapboxSpeech
+
+open class SystemSpeechSynthesizer: NSObject, SpeechSynthesizerController {
+    
+    // MARK: - Properties
+    
+    public var delegate: SpeechSynthesizerDelegate?
+    public var muted: Bool = false // ???
+    public var volume: Float {
+        get {
+            return NavigationSettings.shared.voiceVolume
+        }
+        set {
+            // ?!?!?!
+        }
+    }
```

Comment:
Handling volume and muting with respect to `NavigationSettings.shared.voiceVolume` is not implemented yet